build/yamls/antrea.yml

```diff
@@ -386,7 +386,8 @@ apiVersion: v1
 data:
   antrea-agent.conf: |
     # FeatureGates is a map of feature names to bools that enable or disable experimental features.
-    #featureGates:
+    featureGates:
```

Comment:
I think we cannot modify this file directly. You might update the e2e test YAML to enable TF here: test/e2e/infra/vagrant/push_antrea.sh.
@antoninbas

Reply:
Thanks. I just want to quickly verify e2e with current CI.